### DISCUSSION TITLE
Cargokit BuildTool failed with error:

### DISCUSSION THREAD

**Discussion by rishad13** [fzyzcjy/flutter_rust_bridge#2451] [General]
================================================================================
SEVERE: Cargokit BuildTool failed with error:
SEVERE: --------------------------------------------------------------------------------
SEVERE: External Command: rustup "run" "stable" "cargo" "build" "--manifest-path" "/Users/rishadn/Documents/image_compress_with_rust/rust/Cargo.toml" "-p" "rust_lib_image_compress_with_rust" "--target" "aarch64-linux-android" "--target-dir" "/Users/rishadn/Documents/image_compress_with_rust/build/rust_lib_image_compress_with_rust/build"
SEVERE: Returned Exit Code: 101
SEVERE: --------------------------------------------------------------------------------
SEVERE: STDOUT:
SEVERE: --------------------------------------------------------------------------------
SEVERE: STDERR:
SEVERE: Compiling memchr v2.7.4
SEVERE:    Compiling futures-sink v0.3.31
SEVERE:    Compiling regex-syntax v0.8.5
SEVERE:    Compiling libc v0.2.168
SEVERE:    Compiling quote v1.0.37
SEVERE:    Compiling generic-array v0.14.7
SEVERE:    Compiling pin-project-lite v0.2.15
SEVERE:    Compiling futures-core v0.3.31
SEVERE: error[E0463]: can't find crate for `core`
SEVERE:   |
SEVERE:   = note: the `aarch64-linux-android` target may not be installed
SEVERE:   = help: consider downloading the target with `rustup target add aarch64-linux-android`
SEVERE: 
SEVERE: For more information about this error, try `rustc --explain E0463`.
SEVERE: error: could not compile `regex-syntax` (lib) due to 1 previous error
SEVERE: warning: build failed, waiting for other jobs to finish...
SEVERE: error: could not compile `memchr` (lib) due to 1 previous error
SEVERE: error: could not compile `futures-sink` (lib) due to 1 previous error
SEVERE: error: could not compile `libc` (lib) due to 1 previous error
SEVERE: error: could not compile `pin-project-lite` (lib) due to 1 previous error
SEVERE: error: could not compile `futures-core` (lib) due to 1 previous error
SEVERE: --------------------------------------------------------------------------------
SEVERE: #0      runCommand (package:build_tool/src/util.dart:121:5)
SEVERE: #1      RustBuilder.build (package:build_tool/src/builder.dart:142:5)
SEVERE: <asynchronous suspension>
SEVERE: #2      ArtifactProvider.getArtifacts (package:build_tool/src/artifacts_provider.dart:71:25)
SEVERE: <asynchronous suspension>
SEVERE: #3      BuildGradle.build (package:build_tool/src/build_gradle.dart:35:23)
SEVERE: <asynchronous suspension>
SEVERE: #4      BuildGradleCommand.runBuildCommand (package:build_tool/src/build_tool.dart:65:5)
SEVERE: <asynchronous suspension>
SEVERE: #5      BuildCommand.run (package:build_tool/src/build_tool.dart:37:5)
SEVERE: <asynchronous suspension>
SEVERE: #6      CommandRunner.runCommand (package:args/command_runner.dart:212:13)
SEVERE: <asynchronous suspension>
SEVERE: #7      runMain (package:build_tool/src/build_tool.dart:251:5)
SEVERE: <asynchronous suspension>
SEVERE: --------------------------------------------------------------------------------
SEVERE: BuildTool arguments: [build-gradle]
SEVERE: ================================================================================

**Comment by fzyzcjy** [comment]
For future readers: Discussions are in https://github.com/fzyzcjy/flutter_rust_bridge/issues/2452
